addons/survey/models/survey_survey.py

```diff
@@ -45,6 +45,15 @@ def _get_default_session_code(self):
 
         return False  # could not generate a code
 
+    @api.model
+    def default_get(self, fields_list):
+        result = super().default_get(fields_list)
+        # allows to propagate the text one write in a many2one widget after
+        # clicking on 'Create and Edit...' to the popup form.
+        if not result.get('title') and self.env.context.get('default_name'):
```

Comment:
Should also check that `'title' in fields_list` I think.

Reply:
I'm not sure about the check since we check on the result of the super() call. 'title' will be present in the result if it was in the fields_list no ?

Reply:
Well the docstring of "default_get" mentions:

> Return default values for the fields in ``fields_list``

So I think we should double-check that title was requested by the caller before assigning a value to it.
What do you think?

Reply:
Ok I understand, it's indeed better to double-check ! thanks